README.md

```diff
@@ -38,6 +38,7 @@ Currently it shows:
 * Hostname only displayed when in an SSH session.
 * Username displayed only when it isn't `$LOGNAME`.
 * Username turns red when root.
+* Current Ember.js version, through ember-cli (`🔥`).
```

Comment:
Hamster `🐹` seems great. However Go version uses this emoji too, but I think it will be rarely shown together.